tests/test_cognitive_complexity.py

```diff
@@ -113,16 +113,16 @@ def process_raw_constant(constant, min_word_length):
         raw_camelcase_words = []
         for raw_word in re.findall(r'[a-z]+', constant):  # +1
             word = raw_word.strip()
-            if (  # +2
-                len(word) >= min_word_length  # +4 (2 bool operator sequences * 2 for nesting)
+            if (  # +2 (nesting = 1)
+                len(word) >= min_word_length  # +2 (2 bool operator sequences)
                 and not (word.startswith('-') or word.endswith('-'))
             ):
-                if is_camel_case_word(word):  # +2
+                if is_camel_case_word(word):  # +3 (nesting=2)
```

Comment:
Can't understand how this change is related to sequence of operators bog. Can you make it clear, please?

Reply:
This is just fixing a comment that was incorrect before. The `if` was actually counted as +3 even before #12, and the total in the assert didn't match up with all the comments.

I think this PR fixes the last issue with this particular test, so that's how it ended up in this commit. Happy to force-push the PR branch with this line as a separate commit, will that help?